ui/apps/platform/src/Containers/Vulnerabilities/components/FilterAutocomplete.tsx

```diff
@@ -41,7 +41,7 @@ export type FilterAutocompleteSelectProps = {
     autocompleteSearchContext?:
         | { 'Image SHA': string }
         | { 'Deployment ID': string }
-        | { 'CVE ID': string }
+        | { CVE: string }
```

Comment:
'CVE ID' here is actually incorrect, and worked by coincidence since the previous query was matching on a string prefix. We are passing the CVE _without_ operating system here, so 'CVE ID' is inaccurate.

See the returned fields from the `vulnerabilities` API:
```
						"id": "RHSA-2023:6292#rhel:8",
						"cve": "RHSA-2023:6292",
```